sphinx_rtd_theme/__init__.py

```diff
@@ -60,4 +67,7 @@ def setup(app):
     else:
         app.config.html_add_permalinks = "\uf0c1"
 
+    # Extend the default context when rendering the templates.
+    app.connect("html-page-context", extend_html_context)
```

Comment:
This has been around since sphinx 1.3 by the looks of it. I tested against 1.6 of sphinx (the oldest supported, and it works fine)